TranspotationTicketBooking/Models/Session.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TranspotationTicketBooking.Models
+{
+    public class Session
+    {
+        [Key]
+        public long SId { get; set; }
+        public long BusNo { get; set; }
```

Comment:
In the bus number there are some letters also. So I think this data type is not suitable . it should be change as strings.

Reply:
ok